Whiskey/Tasks/New-WhiskeyProGetUniversalPackage.ps1

```diff
@@ -412,15 +242,34 @@ function New-WhiskeyProGetUniversalPackage
                             return
                         }
 
-                        $robocopyExclude = & { $TaskParameter['Exclude'] ; (Join-Path -Path $destination -ChildPath 'version.json') } 
+                        $robocopyExclude = & {
+                            $taskTempDirectory;
+                            (Join-Path -Path $destination -ChildPath 'version.json');
+                            $TaskParameter['Exclude'];
+                        }
+
                         $operationDescription = 'packaging {0} -> {1}' -f $sourcePath,$destinationDisplay
                         $whitelist = & { 'upack.json' ; $TaskParameter['Include'] }
                     }
 
                     Write-WhiskeyInfo -Context $TaskContext -Message $operationDescription
-                    Invoke-WhiskeyRobocopy -Source $sourcePath.trim("\") -Destination $destination.trim("\") -WhiteList $whitelist -Exclude $robocopyExclude | Write-WhiskeyVerbose -Context $TaskContext
+
+                    $robocopyOutputPath = Join-Path -Path $TaskContext.Temp -ChildPath ('RobocopyOutput.{0}.txt' -f [IO.Path]::GetRandomFileName())
+                    Invoke-WhiskeyRobocopy -Source $sourcePath.trim("\") -Destination $destination.trim("\") -WhiteList $whitelist -Exclude $robocopyExclude | Out-File -FilePath $robocopyOutputPath

```

Comment:
Don't pipe robocopy's output anywhere. Instead, use its `/LOG` parameter to have it write to a log file and then read that file if necessary.